src/Tools/dotnet-monitor/CollectionRules/Options/CollectionRuleDefaultsOptions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options.Triggers;
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.CollectionRules.Options
+{
+    internal class CollectionRuleDefaultsOptions
+    {
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_CollectionRuleDefaultOptions_Egress))]
+        public string Egress { get; set; }
```

Comment:
Something to consider:
- We want this to be relatively simple but over time this single tier of properties could potentially get large. 
- While this system doesn't have to worry about name collisions, we are bringing together unrelated properties. I'm wondering if we should at least split up the defaults into multiple tiers such as Trigger defaults, Action Defaults, etc.